package.json

```diff
@@ -59,7 +59,7 @@
     "*.cjs": "eslint"
   },
   "dependencies": {
-    "@adobe/content-lake-commons": "^1.2.0",
+    "@adobe/content-lake-commons": "file:../content-lake-commons",
```

Comment:
Looks like a residual dev change.

FYI, you can avoid the need to do this with `npm link`. For example, running this command in your `content-lake-extractors-shared` directory:

```
npm link ../content-lake-commons
```

That will point the `@adobe/content-lake-commons` dependency at your local module instead of npm's. You can run `npm install` to undo the change.

Reply:
Thanks @mfrisbey I fixed it and will take the other approach next time.